modules/src/ics03_connection/handler/conn_open_ack.rs

```diff
@@ -168,16 +177,6 @@ mod tests {
             CommitmentPrefix::from(vec![]), // incorrect field
         ));
 
-        // A connection end with correct state & prefix, but incorrect counterparty; exercises
-        // unsuccessful processing path.
-        let mut conn_end_cparty = conn_end_open.clone();
-        conn_end_cparty.set_state(State::Init);
-        conn_end_cparty.set_counterparty(Counterparty::new(
-            client_id.clone(),
-            None, // incorrect field
```

Comment:
This test has been removed as my understanding is that `msg.counterparty_connection_id` should always be a `Some` (which hints that it should not be an `Option`).

Reply:
Now this field is no longer an `Option`.